pkg/kubelet/images/image_manager.go

```diff
@@ -224,3 +372,97 @@ func applyDefaultImageTag(image string) (string, error) {
 	}
 	return image, nil
 }
+
+// isEnsuredBySecret - returns two bools and error
+// 1. pulledBySecret: true if it is in ensured secret pulled image list,
+// the list will clean the image after PullImageSecretRecheckPeriod
+// 2. ensuredBySecret: true if the secret for an auth used to pull an
+// image has already been authenticated through a successful pull request
+// and the same auth exists for this podSandbox/image.
+func (m *imageManager) isEnsuredBySecret(imageRef string, image kubecontainer.ImageSpec, pullSecrets []v1.Secret) (bool, bool, error) {
```

Comment:
can you name the bools in the return so we can easier keep track of which is which?

Reply:
Sure, Addressed

Reply:
Oh okay, Got it !